src/dbi.rs

```diff
@@ -584,6 +597,83 @@ impl<'c> FallibleIterator for DBISectionContributionIter<'c> {
     }
 }
 
+/// See https://github.com/google/syzygy/blob/8164b24ebde9c5649c9a09e88a7fc0b0fcbd1bc5/syzygy/pdb/pdb_data.h#L172
+/// Also see https://www.virtualbox.org/browser/vbox/trunk/include/iprt/formats/codeview.h?rev=93115#L272
+/// This is also known as OMF Segment Map. In the OMF SegmentMap structure, flags and section_type
+/// are a single 16-bit value.
+#[derive(Debug, Copy, Clone)]
+pub struct DBISectionMapItem {
+    /// flags: 0x1 read, 0x2 write, 0x4 execute, 0x8 32-bit
+    pub flags: u8,
+    /// section_type: 0x1 = SEL, 0x2 = ABS, 0x10 = GROUP
+    pub section_type: u8,
```

Comment:
I'd like to see these constants defined in the code (and used where appropriate in `maybe_synthesize_section()`).